ee/vellum_ee/workflows/display/workflows/get_vellum_workflow_display_class.py

```diff
@@ -10,6 +11,24 @@
     from vellum_ee.workflows.display.workflows import BaseWorkflowDisplay
 
 
+def _ensure_display_module_imported(workflow_class: Type[WorkflowType]) -> None:
+    """
+    Best-effort import of the workflow's display module to ensure any custom
+    WorkflowDisplay subclass is registered before we look it up in the registry.
+
+    This allows workflows to work without a display/__init__.py file that
+    re-exports from .workflow and .nodes.
+    """
+    module_name = workflow_class.__module__
+    if module_name.endswith(".workflow"):
+        root = module_name[: -len(".workflow")]
```

Comment:
DevinAI use `re.sub` to be very exact about trimming `.workflow` off the end of `module_name` to produce `root`